src/utils/index.ts

```diff
@@ -2,7 +2,7 @@
  * Lowercases and removes spaces from a string, so that it can be used as a slug.
  */
 export const slugify = (text: string): string => {
-  return text.toLowerCase().replace(/[^\w\-_]+/g, ''); // remove all non-word characters
+  return text.toLowerCase().replace(/[\W_]/g, ''); // remove all non-word characters
```

Comment:
I think realistically many uses cases for a "slug" (e.g. `organic-blueberries` in https://www.goodeggs.com/sfbay/ladybug/organic-blueberries/5f173074927788000eb3c00c) _do_ want to preserve dashes and underscores, which was the intent here. However, since it seems like a bit of a pain to change the behavior just for producer slugs and this doesn't really matter (we just want realistic test data), I'm OK with this solution.